ChangeLog.d/9126.txt

```diff
@@ -0,0 +1,5 @@
+Default behavior changes
+   * In a PSA-client-only build (i.e. MBEDTLS_PSA_CRYPTO_CLIENT &&
+     !MBEDTLS_PSA_CRYPTO_C) no built-in support will be provided for
```

Comment:
I still find “no built-in support will be provided” to be unclear and potentially misleading. You _can_ build with local legacy crypto: the change is that you can now also build without. How about this?

> In a PSA-client-only build (i.e. MBEDTLS_PSA_CRYPTO_CLIENT && !MBEDTLS_PSA_CRYPTO_C), do not automatically enable local crypto when the corresponding PSA mechanism is enabled, since the server provides the crypto. Fixes #9126.